src/components/tab-bar/tab-bar.tsx

```diff
@@ -21,21 +20,21 @@ const { TAB_ACTIVATED_EVENT } = strings;
     shadow: true,
 })
 export class TabBar {
-    @Element()
-    private element: HTMLElement;
-
     /**
      * List of tabs to display
      */
     @Prop()
     public tabs: Tab[];
 
-    private mdcTabBar: MDCTabBar;
-    private setupMdc = false;
-
     @Event()
     private changeTab: EventEmitter<Tab>;
 
+    @Element()
+    private host: HTMLElement;
+
+    private mdcTabBar: MDCTabBar;
+    private setupMdc = false;
```

Comment:
I think it's better to have external stuff above the internal stuff. It makes it easier to see the important stuff.